core/components/atoms/selectionCard/SelectionCard.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { BaseHtmlProps, BaseProps } from '@/utils/types';
+import { useMultiSelect, useSingleSelect } from './hooks';
+
+type ClickEventType = React.MouseEvent<HTMLDivElement> | React.KeyboardEvent;
+
+export interface SelectionCardProps extends BaseProps, BaseHtmlProps<HTMLDivElement> {
+  /**
+   * Element to be rendered inside card item
+   */
+  children: React.ReactNode;
+  /**
+   * Provide unique name to each card when used in group
+   */
+  name?: string;
+  /**
+   * Disable the Selection Card
+   */
+  disabled?: boolean;
+  /**
+   * Handler to be called when Selection card is clicked
+   */
+  onClick?: (e: ClickEventType, name?: string) => void;
```

Comment:
Please use value as second parameter in stead of name. 
Please don't use "e" or any single letter for variables and parameters, use full name like event, onClickEvent etc.